const-directives.md

```diff
@@ -2,8 +2,210 @@
 
 ## Table of Contents
 
-1. [Ambient occlusion level](#ambient-occlusion-level)
+1. [Texture Format](#texture-format)
+2. [Texture Clear](#texture-clear)
+3. [Texture Mipmaps](#texture-mipmaps)
+4. [Shadow Texture Mipmaps](#shadow-texture-mipmaps)
+5. [Instance Count](#instance-count)
+6. [Ambient occlusion level](#ambient-occlusion-level)
+7. [Super sampling level](#super-sampling-level)
 
+## Texture Format
+
+This directive allows a shader pack to specify the format and precision of a colortex or shadowcolor color attachment.
```

Comment:
This section is very useful! I would recommend linking to further reading like https://www.khronos.org/opengl/wiki/Small_Float_Formats as well for people needing to go even deeper for whatever reason.

Reply:
btw if you didn't see it, I added the link further down where those specific formats are discussed (line 45)